src/bio/volume_io.cpp

```diff
@@ -89,6 +89,12 @@ VolumeIo::~VolumeIo(void)
 {
 }
 
+void
+VolumeIo::UpdateSectorRba(uint32_t sectorCount)
+{
+    sectorRba += sectorCount;
+}
```

Comment:
original code에 있는 다음 코드랑 의미는 다른데 메서드 명은 비슷해서요, 혹시 메서드명을 조금 다르게 지어야 할까요??
```c++
void
VolumeIo::SetSectorRba(uint64_t inputSectorRba)
{
    if (unlikely(_IsInvalidSectorRba(inputSectorRba)))
    {
        POS_TRACE_ERROR(EID(UBIO_INVALID_RBA),
            "Invalid RBA for Ubio");
        return;
    }

    sectorRba = inputSectorRba;
}
```

Reply:
네 요건 add 하는 코드인데, 이름을 좀 더 명확하게 바꾸도록 하겠습니다.

Reply:
GPT님께서 아래처럼 답변을 주시는 군요. 

The functionname "UpdateSectorRba" seems fine and accurately describes what the function does. However, if you wanted to follow naming conventions, you might consider using a name like "UpdateSectorRbaCount" or "AddToSectorRbaCount". This will make the purpose of the function clearer to other programmers who might be reading your code.

Reply:
AddToSectorRbaCount 괜찮아 보입니다. ㅋㅋ

Reply:
오... 메서드명 제안도 해주는군요!